pages/api/project.ts

```diff
@@ -6,7 +6,7 @@ type ProjectType = {
   author: string;
```

Comment:
if possible please make it like user api. inside projects folder add [projectid].ts and index.ts because we need to display project details in project details view

Reply:
I haven't finish the project part yet, In a while, I'll push updates for project API